### PR TITLE
[WIP] V2021.10 nvec paz00

### PR DESCRIPTION
**Pull request by kwizart**
This is a rebase of my tree from 2021.10 nvec paz00 driver.

Known not to work is EFI boot on tegra20 (works for tegra124).
please review.

### CODE REVIEW

File: drivers/i2c/tegra_nvec.c [hunk 1]
```diff
@@ -410,7 +411,7 @@ static int nvec_probe(struct udevice *dev)
 {
 	struct nvec_t *nvec = dev_get_priv(dev);
 	const void *blob = gd->fdt_blob;
-	int node = dev->node.of_offset;
+	int node = dev->node_.of_offset;
```
Comment (kwizart) [inline drivers/i2c/tegra_nvec.c]: This should probably be deleted (can be squashed with the later patch).

File: drivers/i2c/tegra_nvec.c [hunk 2]
```diff
@@ -425,7 +421,7 @@ static int nvec_probe(struct udevice *dev)
 
 	debug("NVEC initialization...\n");
 
-	if (nvec_decode_fdt(blob, node, nvec_data)) {
+	if (nvec_decode_fdt(nvec_data) {
```
Comment (kwizart) [inline drivers/i2c/tegra_nvec.c]: Missing bracket,  will fix...

**Comment by kwizart** [comment]
It fails to build from source, I will revisit in the next days. I need to recover a known good and tested version...

**Comment by clamor-s** [comment]
> It fails to build from source, I will revisit in the next days. I need to recover a known good and tested version...

Sources were updated. May you rebase your PR please?

**Comment by kwizart** [comment]
Sure, I hope to update this over the week-end.

**Comment by clamor-s** [comment]
> Sure, I hope to update this over the week-end.

Rebase your changes please.

**Comment by kwizart** [comment]
The nvec fork for paz00 is currently maintained at 
https://gitlab.com/ac100/u-boot-ac100-exp/-/merge_requests/2
I'm seeking review from @marvintwentyfour in a first step.

I've rebased from grate until then.

**Comment by kwizart** [comment]
Closing as I've submitted a new PR with a new base branch
https://github.com/grate-driver/u-boot/pull/2

**Pull request closed without merging**